test/services/licences/fetch-charge-versions.service.test.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const ChargeVersionHelper = require('../../support/helpers/charge-version.helper.js')
+const ChangeReasonHelper = require('../../support/helpers/change-reason.helper.js')
+
+// Thing under test
+const FetchChargeVersionsService =
+  require('../../../app/services/licences/fetch-charge-versions.service.js')
+
+describe('Fetch charge versions for a licence', () => {
```

Comment:
Our convention is to name the top-level 'describe' after the thing being tested.

We believe it makes it easier to match up the test output to the test file.

```suggestion
describe('Fetch Charge Versions service', () => {
```